modules/nlu/src/backend/language/language-provider.ts

```diff
@@ -438,6 +503,17 @@ export class RemoteLanguageProvider implements LanguageProvider {
     // we restore original chars and casing
     return tokenUtterances.map((tokens, i) => restoreOriginalUtteranceCasing(tokens, utterances[i]))
   }
+
+  private computeVersionHash = () => {
+    const { nluVersion, langServerVersion } = this._state
+
+    const omitPatchNumber = (v: string) => `${semver.major(v)}.${semver.minor(v)}.0`
+    const hashContent = `${omitPatchNumber(nluVersion)}:${omitPatchNumber(langServerVersion)}`
```

Comment:
is there a version in lang-server ?

Reply:
There is now do you prefer to only use the nlu-version instead?